Tests/scripts/uninstall_packs_and_reset_bucket.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# exit on errors
+set -e
+
+XSIAM_SERVERS_PATH=$(cat xsiam_servers_path)
```

Comment:
If this script is hardcoded to handle only XSIAM, it should be noted in the script's name. Why doesn't XSOAR have this same behavior? why don't we uninstall packs from the XSOAR servers?

Reply:
XSOAR don't have the same behaviour, since in xsoar we destroying xsoar instances. In xsiam we are re-using xsiam instances.